packages/storybook/.storybook/shared/config.js

```diff
@@ -0,0 +1,6 @@
+import { addDecorator } from "@storybook/react";
+import { withInfo } from "@storybook/addon-info";
+import { withKnobs } from "@storybook/addon-knobs/react";
+
+addDecorator((story, context) => withInfo({ header: true })(story)(context));
```

Comment:
In a future PR, lets remove the global `withInfo`, so we can add proper docs for each component.